sdk/lib/crt/except/i386/__CxxFrameHandler3.s

```diff
@@ -0,0 +1,26 @@
+
+
+#include <asm.inc>
+
+.code
+.align 4
```

Comment:
What does this ".align 4" affect, in this code?

Reply:
The alignment.